src/loaddb/load_worker_manager.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation. All rights reserved by Search Solution.
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ */
+
+/*
+ * load_manager.cpp - entry point for loaddb manager
+ */
+
+#include "load_worker_manager.hpp"
+
+namespace cubload
+{
+  static loaddb_worker_context_manager *g_loaddb_wp_context_manager;
```

Comment:
`loaddb` prefix is not necessary, since they all belong to cubload namespace. better to keep the names shorter.

```suggestion
  static worker_context_manager *g_wp_context_manager;
```